apps/settings/elements/browsing_privacy.html

```diff
@@ -21,6 +21,14 @@ <h1 data-l10n-id="browsingPrivacy-header"></h1>
             <details data-l10n-id="privateBrowsingDefaultDetails"></details>
           </gaia-switch>
         </li>
+        <li>
+          <gaia-switch name="privacy.trackingprotection.enabled">
+            <label data-l10n-id="trackingProtection"></label>
+            <details data-l10n-id="trackingProtectionDetails"></details>
+            <a data-l10n-id="learn-more"
```

Comment:
I'll look at the UI of this later, but it feels like we may want this inside of `<details>` node. Is that possible?